eggplant/membership/tests.py

```diff
@@ -162,7 +158,7 @@ def test_send_invitation(self):
         expected = 'Invitation has been send to {}'.format(invited_email)
         self.assertContains(response, expected, 1, 200)
 
-        self.assertEqual(len(mail.outbox), 1)
+        self.assertEqual(len(mail.outbox), 1)  # @UndefinedVariable
```

Comment:
If it is okay with you, I would just _love_ if we keep those editor specific things out of the codebase? Pretty please :heart_eyes: